docsets/Xojo/docset.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Xojo 2023r4",
+    "version": "1.0.0",
+    "archive": "Xojo_2023r4.tgz",
```

Comment:
Don't include the version in the archive name.

Reply:
See my comments above and let me know how you'd like to proceed.